openid-4-verifiable-presentations-1_0.md

```diff
@@ -2836,6 +2831,7 @@ The technology described in this specification was made available from contribut
    
    * clarify DCQL case of `claims` and `claim_sets` being absent
    * add language on client ID and nonce binding for ISO mdocs and W3C VCs
+   * for DC API, always use Origin for binding the response (e.g. in Key Binding JWT `aud`)
```

Comment:
```suggestion
   * for DC API, omit client_id from the request and always use Origin for binding the response (e.g. in Key Binding JWT `aud` and sessionTranscript in mdoc)
```

Reply:
As noted somewhere else, `client_id` is still in the request - committing the other part of your suggestion though:

```suggestion
   * for DC API, always use Origin for binding the response (e.g. in Key Binding JWT `aud` and sessionTranscript in mdoc)
```